documentation/user-interface/code-editor.md

```diff
@@ -2,15 +2,16 @@
 layout: docs
 title: Code Editor
 permalink: /documentation/user-interface/code-editor.html
+checked: false
 ---
 # Code Editor
 
 The **Code Editor** allows you to write and modify [coded tests](../guides/organize-tests.md#coded-test-files). To open the editor, double-click a test file's node or a test's node in the [Explorer](explorer-panel.md) panel. If you create a new test file (see [Write Test Code](../guides/write-test-code.md)), TestCafe Studio automatically opens the file in the editor.
 
 ![Code Editor](../../images/guides/code-editor.png)
 
-You can write coded tests using JavaScript or TypeScript. For information about the API used in tests, see the [Test API](https://devexpress.github.io/testcafe/documentation/test-api/) topic in the open-source TestCafe documentation.
+You can write coded tests using JavaScript or TypeScript. Refer to the [Test API](https://devexpress.github.io/testcafe/documentation/test-api/) topic in the open-source TestCafe documentation for information about the API used in tests.
 
 The **Code Editor** provides the following code edit features: syntax highlight, code completion, parameter hints, bracket matching, go to definition, peek definition and so on.
 
-The editor also provides the ![Run test button](../../images/user-interface/run-target-icon.svg) buttons that allow you to run an entire fixture and individual tests. These buttons are shown in the gutter for lines that define fixtures and tests.
\ No newline at end of file
+The editor also provides the ![Run test button](../../images/user-interface/run-target-icon.svg) buttons that allow you to run a fixture and individual tests. These buttons are shown in the gutter for lines that define fixtures and tests.
```

Comment:
"the gutter for lines that define fixtures and tests" The gutter?

Reply:
Gutter is a term

Reply:
Ok